weather-app/src/main/res/layout/weather.xml

```diff
@@ -10,12 +10,15 @@
     tools:context=".WeatherListActivity"
     >
 
+
   <android.support.design.widget.CoordinatorLayout
       android:id="@+id/content"
       android:layout_width="match_parent"
       android:layout_height="match_parent"
       android:orientation="vertical"
       >
+
```

Comment:
Try to run the autoformat tool, this is going to help the reviewers to focus on code changes and not new empty lines, spaces or style changes.